neural_modelling/src/neuron/plasticity/stdp/synapse_dynamics_stdp_mad_impl.c

```diff
@@ -270,7 +270,7 @@ bool synapse_dynamics_initialise(
 
     // Load timing dependence data
     address_t weight_region_address = timing_initialise(address);
-    if (address == NULL) {
+    if (weight_region_address == NULL) {
```

Comment:
This change was an absolute howler. Fortunately, I think the compiler was proving in all cases that the branch couldn't be taken, but it's still absurd.